go.mod

```diff
@@ -1 +1,3 @@
 module github.com/elazarl/goproxy
+
+require github.com/elazarl/goproxy/ext v0.0.0-20190711103511-473e67f1d7d2
```

Comment:
Why do we need this? main goproxy module shouldn't have dependency, should it?

Reply:
`github.com/elazarl/goproxy/ext` has its own `go.mod` file, which means its a standalone Go module. Since this module is consumed by your `github.com/elazarl/goproxy` package, a dependency is formed. I only performed this change because the current `go.mod` was invalid.

If you have no plans to use `github.com/elazarl/goproxy/ext` as a separate package, you could remove the `go.mod` file. But that's outside the scope of this PR.

Reply:
I understand. goproxy shouldn't depend on `goproxy/ext`. But as you said, this is a mistake for another PR.
Thanks.